Packs/AutoFocus/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### AutoFocus Feed
+-
```

Comment:
```suggestion
Improved parameter description.
```

Reply:
@orhovy You can apply Bar's suggestions to all of the release notes.